Source/ChartboostAdapter.swift

```diff
@@ -122,11 +122,11 @@ final class ChartboostAdapter: PartnerAdapter {
     /// A default implementation is provided that returns `nil`.
     /// Only implement if the partner SDK provides its own list of error codes that can be mapped to Chartboost Mediation's.
     /// If some case cannot be mapped return `nil` to let Chartboost Mediation choose a default error code.
-    func mapSetUpError(_ error: Error) -> ChartboostMediationError.Code? {
-        guard let error = error as? CHBStartError else {
+    func mapSetUpError(_ error: StartError) -> ChartboostMediationError.Code? {
```

Comment:
The method defined by PartnerAdapter is `func mapSetUpError(_ error: Error) -> ChartboostMediationError.Code? `. By changing the type to `StartError` in the method signature I think we'll end up with this method never getting called, since it's not the one the SDK expects.
Same for the rest

Reply:
Ok, I fixed that and forced a load error to test that `mapLoadError` was actually getting called now.